src/pages/iou/MoneyRequestDescriptionPage.js

```diff
@@ -0,0 +1,89 @@
+import React, {Component} from 'react';
+import {View} from 'react-native';
+import {withOnyx} from 'react-native-onyx';
+import PropTypes from 'prop-types';
+import TextInput from '../../components/TextInput';
+import withLocalize, {withLocalizePropTypes} from '../../components/withLocalize';
+import ScreenWrapper from '../../components/ScreenWrapper';
+import HeaderWithCloseButton from '../../components/HeaderWithCloseButton';
+import Form from '../../components/Form';
+import ONYXKEYS from '../../ONYXKEYS';
+import styles from '../../styles/styles';
+import Navigation from '../../libs/Navigation/Navigation';
+import compose from '../../libs/compose';
+import * as IOU from '../../libs/actions/IOU';
+
+const propTypes = {
+    ...withLocalizePropTypes,
+
+    /** Onyx Props */
+    /** Holds data related to Money Request view state, rather than the underlying Money Request data. */
+    iou: PropTypes.shape({
+        comment: PropTypes.string,
+    }),
+};
+
+const defaultProps = {
+    iou: {
+        comment: '',
+    },
+};
+
+class MoneyRequestDescriptionPage extends Component {
+    constructor(props) {
+        super(props);
+
+        this.updateComment = this.updateComment.bind(this);
+    }
+
+    /**
+     * Sets the IOU comment by saving it to Onyx.
+     *
+     * @param {Object} value
+     * @param {String} value.iouComment
+     */
+    updateComment(value) {
+        IOU.setMoneyRequestDescription(value.iouComment);
+        Navigation.goBack();
+    }
+
+    render() {
+        return (
+            <ScreenWrapper includeSafeAreaPaddingBottom={false}>
+                <HeaderWithCloseButton
+                    title={this.props.translate('common.description')}
+                    shouldShowBackButton
+                    shouldShowCloseButton={false}
+                    onBackButtonPress={Navigation.goBack}
+                />
+                <Form
+                    style={[styles.flexGrow1, styles.ph5]}
+                    formID={ONYXKEYS.FORMS.IOU_DESCRIPTION_FORM}
+                    onSubmit={this.updateComment}
+                    submitButtonText={this.props.translate('common.save')}
+                    validate={() => ({})}
+                    enabledWhenOffline
+                >
+                    <View style={styles.mb4}>
+                        <TextInput
+                            inputID="iouComment"
+                            name="iouComment"
+                            defaultValue={this.props.iou.comment}
+                            label={this.props.translate('iOUConfirmationList.whatsItFor')}
```

Comment:
I feel like an example placeholder might work well here.

Reply:
The label ends up moving to the top of the text field once you start inputting, so an example placeholder wouldn't feel correct in that case:
<img width="365" alt="Screenshot 2023-04-12 at 9 39 11 AM" src="https://user-images.githubusercontent.com/4741899/231524591-15dfa31b-abfe-47f6-94ed-14e7ec0a8cb8.png">